public/stylesheets/login-local.css

```diff
@@ -0,0 +1,32 @@
+body {
+    padding-top: 90px;
```

Comment:
related to what Austin said below: we might want to have some baseline definition file for stuff like typeface, and move things like padding and text alignment to either individual component files or to a separate style file. I'm not sure what form is better though, especially given the possibility of breaking everything by throwing in Bootstrap stuff